test/e2e/flux_test.go

```diff
@@ -311,8 +311,8 @@ func TestDockerInstallGithubFluxDuringUpgrade(t *testing.T) {
 	runUpgradeFlowWithFlux(
 		test,
 		v1alpha1.Kube122,
-		framework.WithFluxGithub(),
-		framework.WithClusterUpgrade(),
+		framework.WithFluxGithub(api.WithFluxConfigName(framework.DefaultFluxConfigName)),
+		framework.WithClusterUpgrade(api.WithGitOpsRef(framework.DefaultFluxConfigName, v1alpha1.FluxConfigKind)),
```

Comment:
why not implement this in `WithFluxGithub`?

Reply:
yeah, that would be ideal. However, at the moment, the WithClusterUpgrade seems to override the GitOpsRef name with its own fillers derived from the initial cluster spec. So this is an attempt to verify that is what's happening and get the names aligned, so that we don't end up with a GitOpsRef that points to nothing